src/components/ContactUs.jsx

```diff
@@ -1,4 +1,33 @@
+import '../reset.css';
```

Comment:
I think it's better to include it only once (in the `App` component for example)